internal/controllers/clusterextension_controller_test.go

```diff
@@ -531,8 +531,14 @@ func TestClusterExtensionInstallationFailedWatcherFailed(t *testing.T) {
 	t.Log("By checking the expected installed conditions")
 	installedCond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeInstalled)
 	require.NotNil(t, installedCond)
-	require.Equal(t, metav1.ConditionFalse, installedCond.Status)
-	require.Equal(t, ocv1alpha1.ReasonInstallationFailed, installedCond.Reason)
+	require.Equal(t, metav1.ConditionTrue, installedCond.Status)
+	require.Equal(t, ocv1alpha1.ReasonSuccess, installedCond.Reason)
+
+	t.Log("By checking the expected managed conditions")
```

Comment:
Is there a way to remediate this state? I assume it's not a critical state because the CE is still installed, but we just can't respond to any changes to the managed resources. Is that correct?

Reply:
That's correct. 

I think the most common issue that will happen here will be permission related. With the new setup introduced by this PR, any informer errors result in a requeue of the `ClusterExtension` with exponential backoff.

Remediation for the permission error is manual user intervention to assign the missing permissions to the `ServiceAccount` referenced in `ClusterExtension.Spec.ServiceAccount.Name`